client/js/transports/small-webrtc.mdx

```diff
@@ -28,7 +28,7 @@ const pcClient = new PipecatClient({
 });
 
 await pcClient.connect({
-  endpoint: "https://your-server/connect",
+  webrtc_url: "/api/offer", // Your WebRTC signaling server endpoint
```

Comment:
here do we want to show the camelCase version?

Reply:
yup working on finding all those...